ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -112,25 +140,36 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal {
         case alwaysHidden
     }
 
-    public typealias TokenSetKeyType = EmptyTokenSet.Tokens
-    public var tokenSet: EmptyTokenSet = .init()
+    public typealias TokenSetKeyType = NavigationBarTokenSet.Tokens
+    public lazy var tokenSet: NavigationBarTokenSet = .init(style: { [weak self] in
+        self?.style ?? NavigationBar.defaultStyle
+    })
 
     static let expansionContractionAnimationDuration: TimeInterval = 0.1 // the interval over which the expansion/contraction animations occur
 
     private static var defaultStyle: Style = .primary
 
+    // These two constants are based on OS default values
+    static let systemHeight: CGFloat = 44
+    static let compactSystemHeight: CGFloat = 32
+    static let extraPaddingForCompactBar: CGFloat = systemHeight - compactSystemHeight
+
     private struct Constants {
-        static let systemHeight: CGFloat = 44
         static let normalContentHeight: CGFloat = 44
         static let expandedContentHeight: CGFloat = 48
 
-        static let leftBarButtonItemLeadingMargin: CGFloat = 8
-        static let rightBarButtonItemHorizontalPadding: CGFloat = 10
+        static let leftBarButtonItemLeadingMargin: CGFloat = GlobalTokens.spacing(.size80)
+        static let leftBarButtonItemTrailingMargin: CGFloat = GlobalTokens.spacing(.size80)
+        static let rightBarButtonItemHorizontalPadding: CGFloat = GlobalTokens.spacing(.size100)
```

Comment:
Shouldn't these also be in the tokenSet?

Reply:
Not exactly a high-priority item, but sure.